CMakeLists.txt

```diff
@@ -343,7 +343,23 @@ if(CELERITAS_USE_MPI)
 endif()
 
 if(CELERITAS_USE_OpenMP)
-  find_package(OpenMP REQUIRED)
+  if(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
+    # MSVC requires LLVM OpenMP support to provide 3.0 or higher
```

Comment:
I think the `llvm` option is still experimental and only supports a subset of features from newer specs.

Reply:
Yeah. It supports the `atomic capture` keyword in theory but breaks in practice and self-reports as 2.0 anyway.